tutor/src/models/course/scores/task-result.js

```diff
@@ -41,19 +42,18 @@ export default class TaskResult extends BaseModel {
 
   @computed get progress() {
     if (!this.step_count){ return null; }
-    const count = this.completed_on_time_step_count + this.completed_accepted_late_step_count;
-    return count / this.step_count;
+    return this.completedStepCount / this.step_count;
+  }
+
+  @computed get isHomework() {
+    return this.type === 'homework';
   }
 
   @computed get lateStepCount() {
-    if (this.type === 'homework') {
-      return this.completed_exercise_count -
-             this.completed_on_time_exercise_count -
-             this.completed_accepted_late_exercise_count;
+    if (this.isHomework) {
+      return this.completed_exercise_count - this.completed_on_time_exercise_count;
     }
-    return this.completed_step_count -
-           this.completed_on_time_step_count -
-           this.completed_accepted_late_step_count;
+    return this.completed_step_count - this.completed_on_time_step_count;
```

Comment:
This works, but changes what the method does (i.e it will no longer take into account whether or not the instructor accepted late work). Is this desired?

Reply:
This method is used to determine if there is late work that could be accepted.  It should be > 0 if there's new work that has not yet been accepted.

Reply:
I think you might want `this.completed_step_count - this. completedStepCount` in that case?

Reply:
Ahah, ok I think I see what you're saying.  That makes sense.  Sorry for the confusion